internal/pkg/argocd/argocd.go

```diff
@@ -474,7 +474,7 @@ func generateDiffOfAComponent(ctx context.Context, commentDiff bool, componentPa
 		app, err := ac.app.Get(ctx, &appNameQuery)
```

Comment:
Just in case someone else was looking at this comment:
 > what's the difference between the app we have and the app we get though
We do the 2nd App "get"  to `HardRefresh` the resources, we can't do it on the first app get as we might pull multiple apps and HardRefresh is a very expensive and slow operation(git clone/helm template ...)